### PR TITLE
Feature/update label

### PR DESCRIPTION
**Pull request by Jamie-BitFlight**
## What
Update the version of label to 0.11.1
Update the example to show how one label module can be used with the modules.

## Why
Allows passing the label module context into the VPC module, also adds all other current label module variables.

### CODE REVIEW

**Comment by Jamie-BitFlight** [comment]
@osterman can you check in on this?

File: examples/basic/main.tf [hunk 1]
```diff
@@ -1,20 +1,30 @@
 module "vpc" {
-  source     = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
-  namespace  = "${var.namespace}"
-  stage      = "${var.stage}"
-  name       = "${var.name}"
+  # source     = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
```
Comment (aknysh) [inline examples/basic/main.tf]: remove commented code

Comment (aknysh) [reply examples/basic/main.tf]: @Jamie-BitFlight can you please address the comment ^

**Comment by maximmi** [comment]
@Jamie-BitFlight could you please address the comments

**Comment by Gowiem** [comment]
This is pretty old, includes a number of conflicts, and contributor isn't responding. Closing out!

**Pull request closed without merging**